src/data/markdown/translated-guides/en/03 Using k6 browser/02 Running browser tests.md

```diff
@@ -25,55 +25,69 @@ To run a simple local script:
   <CodeGroup labels={["script.js"]} lineNumbers={[true]}>
 
   ```javascript
-  import { chromium } from 'k6/experimental/browser';
+  import { browser } from 'k6/experimental/browser';
+
+  export const options = {
+    scenarios: {
+      ui: {
+        executor: 'shared-iterations',
+        options: {
+          browser: {
+            type: 'chromium',
+          },
+        },
+      },
+    },
+    thresholds: {
+      checks: ["rate==1.0"]
+    }
+  }
 
   export default async function () {
-    const browser = chromium.launch({
-      headless: false,
-      timeout: '60s', // Or whatever time you want to define
```

Comment:
We should not remove this as part of the examples. One of the questions we are asked is how to set a timeout for their tests. Perhaps, we can add an example of how to use the new environment variables, especially `K6_BROWSER_HEADLESS` and `K6_BROWSER_TIMEOUT`

Reply:
👍 @ankur22 I didn't see the metrics like `browser_web_vital_cls_good` that ends with `_good`. Is it expected?

Reply:
> I couldn't see the metrics like browser_web_vital_cls_good that ends with _good when I ran the example in this PR. Is it expected? I guess this rating feature hasn't been released yet, that's why.

The rating metrics have been moved and now are set as labels in the WebVitals metrics. See https://github.com/grafana/xk6-browser/pull/915.